lib/virtual-net/src/host.rs

```diff
@@ -13,20 +13,21 @@ use std::ptr;
 use std::sync::Mutex;
 use std::task::{Context, Poll, RawWaker, RawWakerVTable, Waker};
 use std::time::Duration;
+use tokio::runtime::Handle;
 use tokio::sync::mpsc;
 #[allow(unused_imports, dead_code)]
 use tracing::{debug, error, info, trace, warn};
 
 #[derive(Debug)]
 pub struct LocalNetworking {
-    // Make struct internals private.
-    // Can be removed once some fields are added (like permissions).
-    _private: (),
+    handle: Handle,
```

Comment:
As above for the FS, this seems very hacky.

Reply:
As above